external/iotivity/iotivity_1.2-rel/resource/csdk/stack/src/oicgroup.c

```diff
@@ -819,7 +819,7 @@ OCStackResult BuildStringFromActionSet(OCActionSet* actionset, char** desc)
     actionTypeStr = (char *)OICMalloc(1024);
     if(actionTypeStr != NULL)
     {
-        sprintf(actionTypeStr, "%ld %u", actionset->timesteps, actionset->type);
+        snprintf(actionTypeStr, sizeof(actionTypeStr), "%ld %u", actionset->timesteps, actionset->type);
```

Comment:
`sizeof(actionTypeStr)`==`sizeof(char *)`, I think you mean this:
 ```suggestion
        snprintf(actionTypeStr, 1024, "%ld %u", actionset->timesteps, actionset->type);
```